code/pizzabot.js

```diff
@@ -1,7 +1,138 @@
+    document.getElementById("progressOne").style.background = "#01DC3E";
+
+    displayOrderQuantityBox()
+}
+
+// Counting-function to select amount of pizzas
+let quantityFunction = (event) => {
+    event.preventDefault();
+
+    // Get quantity from user input-counter 
+    let orderQuantity = document.getElementById("quantityUser").value;
+
+    // The box which confirms Pizza-sort + price + cooking-time
+    const confirmationBox = document.getElementById("confirmationBox");
+    
+    // Total cost-function
+    let orderTotal;
+    let calculateTotalCost = () => {
+        
+        // Conditional statement given other price for kebab pizza
+        let otherPrice = () => {
+            if (orderName === kebab)
+            {
+                pizzaPrice = 100;
+            } 
+            else {
+                pizzaPrice = 80;
+            }
+        }
+        
+        otherPrice() 
+
+        orderTotal = orderQuantity * pizzaPrice;
+    }
+
+    // Total cooking-time-function
+    let cookingTime;
+
+    let calculateCookingTime = () => {
+        if (orderQuantity < 3) {
+            cookingTime = 10;
+        } else if (orderQuantity < 6) {
+            cookingTime = 15;
+        } else {
+            cookingTime = 20;
+        }
+    }
+    
+    calculateTotalCost()
+    calculateCookingTime()
+
+    // Confirmation message
+    document.getElementById("topBarCount").innerHTML = `${orderQuantity} x`; 
+    document.getElementById("progressTwo").style.background = "#01DC3E";
+    document.getElementById("confirmation").innerHTML = `Great, I'll get started on your ${orderName}(s) right away, it will cost ${orderTotal} kr. The pizza(s) will take ${cookingTime} minutes.`;
+    
+    confirmationBox.classList.remove("invisible");
+    confirmationBox.classList.add("visible");
+
+    //Count-down timer gets visible
+    timerBox.classList.remove("invisible");
+    timerBox.classList.add("visible");
+
+    //Order Summary   
+    document.getElementById("count").innerHTML = `${orderQuantity}`;
+    document.getElementById("pizza").innerHTML = `${orderName}`;
+    document.getElementById("price").innerHTML = `${pizzaPrice} kr`;
+    document.getElementById("total").innerHTML = `Total: ${orderTotal} kr`;
+    document.getElementById("topBarTotal").innerHTML = ` = ${orderTotal} kr`;
+    
+    //Order Summary gets visible
+    const orderSummaryBox = document.getElementById("orderSummaryBox");
+    orderSummaryBox.classList.remove("invisible");
+    orderSummaryBox.classList.add("visible");
+
+    //Count-down timer function
+    let startingMinutes = cookingTime;
+    let time = startingMinutes * 60;
+
+    const countdownEl = document.getElementById("time");
+
+    const updateCountdown = () => {   
+        const minutes = Math.floor(time / 60);
+        let seconds = time % 60;
+
+        seconds = seconds < 10 ? '0' + seconds : seconds;
+    
+        countdownEl.innerHTML = `${minutes}:${seconds}`;
+        time--;
```

Comment:
Your timer is counting time down even after reaching 00:00 - that leads to time lower than 0 later on

Reply:
@EmOlsz We are totally aware of that but we didn't get it to work unfortunately.. Do you have any tip? :)